docs/source/api/server_server_capabilities.rst

```diff
@@ -0,0 +1,223 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-server-server-capabilities:
+
+******************************
+``server_server_capabilities``
+******************************
+
```

Comment:
This endpoint was added in version 1.4, use the `.. versionadded::` directive to indicate that.